### ISSUE TITLE
cpplint errors in b1dc9f5

### ISSUE THREAD

**Issue by larkost** [rethinkdb/rethinkdb#3763]
There are three `cpplint` errors in `next` at the moment:
```
src/clustering/administration/jobs/report.hpp:149:  TODO(my_username) should be followed by a space  [whitespace/todo] [2]
src/concurrency/new_mutex.hpp:50:  Single-parameter constructors should be marked explicit.  [runtime/explicit] [5]
src/rdb_protocol/protocol.hpp:180:  Single-parameter constructors should be marked explicit.  [runtime/explicit] [5]
```
@VeXocide @mlucy @Tryneus 
A simple `git blame` says that three different people last touched the lines (@VeXocide, @mlucy, and @Tryneus respectively), so this one is a tossup.

**Comment by VeXocide** [comment]
I believe @Tryneus actually fixed the `TODO` as part of #3662 thus that should be gone soon regardless.

**Comment by Tryneus** [comment]
Also, I'm getting rid of the `new_mutex_acq_t` in the same branch, I'll just take all of these.

**Comment by danielmewes** [comment]
@Tryneus Are you replacing `new_mutex_acq_t` by something else?

**Comment by Tryneus** [comment]
@danielmewes I ran into a problem with `new_mutex_acq_t` resulting in two coroutines having the lock - the problem went away using `new_mutex_in_line_t` directly.  Since what I was working with was the only code in `next` using `new_mutex_acq_t`, I thought it would be fine to remove it.  @timmaxw apparently is using it in his branch, so I'll unremove it and try to debug it.

**Comment by danielmewes** [comment]
@Tryneus Yeah if `new_mutex_acq_t` caused the mutex itself to misbehave that could mean that there's a bug further down?

**Comment by Tryneus** [comment]
This has been fixed and merged to `next` alongside the fix for #3662, in commit 4cdb9ba.  Will be in release 2.0.
